src/bindings/cpp/include/keyset.hpp

```diff
@@ -31,7 +31,9 @@ class KeySet
 	inline KeySet(ckdb::KeySet *k);
 	inline KeySet(const KeySet &other);
 
-	inline explicit KeySet(size_t alloc, va_list ap);
+#if defined(__amd64__) || defined(__x86_64__) || defined(__i386__)
```

Comment:
Hello,

manuel wrote:
> I have no problem with removing the va constructor. Shall I remove it right
> away and rebase?

Actually, I still prefer to distinguish by useless argument. C++11 also uses 
this technique, see std::piecewise_construct_t. The advantage is that it does 
not impose any significant runtime overhead, opposed to creating and 
destroying a KeySet that would happen with fromVA. When a static object is 
provided, its also easy to use.

But if you think that from the bindings perspective fromVA is the better 
solution you can rebase right away ;)

best regards,